frontend/src/app/organization/organization-list/organization-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+    <h1 class="page-header">Lista organizacji</h1>
+    <div *ngIf="organizations.length">
+        <table class="table table-striped">
+            <thead>
+            <tr>
+                <th>Nazwa</th>
+                <th>Adres</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr *ngFor="let organization of organizations">
```

Comment:
I would say it's probably better to go with some data-tables from the beginning. Just a little effort, and it will not have to be re-done soon. Pagination would be useful, I guess, pretty quickly.

That could lead to some more sane application layout: there is one "smart", routable organizations view component, and the table itself could be the "dumb" one with @Input (and @Output if needed).